templates/CRM/Event/Form/Participant.tpl

```diff
@@ -274,25 +271,26 @@
 
   {/if}
 
-
-<script type="text/javascript">
-  {literal}
-
-  sendNotification();
-  function sendNotification() {
-    var notificationStatusIds = {/literal}"{$notificationStatusIds}"{literal};
-    notificationStatusIds = notificationStatusIds.split(',');
-    if (cj.inArray(cj('.crm-participant-form-block-status_id select#status_id option:selected').val(), notificationStatusIds) > -1) {
-      cj("#notify").show();
-      cj("#is_notify").prop('checked', false);
-    }
-    else {
-      cj("#notify").hide();
-      cj("#is_notify").prop('checked', false);
+  {if $action NEQ 8}
+  <script type="text/javascript">
+    {literal}
+
+    sendNotification();
+    function sendNotification() {
+      var notificationStatusIds = {/literal}"{$notificationStatusIds}"{literal};
+      notificationStatusIds = notificationStatusIds.split(',');
+      if (cj.inArray(cj('.crm-participant-form-block-status_id select#status_id option:selected').val(), notificationStatusIds) > -1) {
+        cj("#send_notification").show();
+        cj("#is_notify").prop('checked', false);
+      }
+      else {
+        cj("#send_notification").hide();
+        cj("#is_notify").prop('checked', false);
```

Comment:
Minor note: Since it sets checked to false either way could take it out of the `if`

Reply:
Actually, I think this one was wrong. We shouldn't be unchecking when we are showing, since we could have triggered on a change in the select from one status in the array to another that's also in the array.